### PR TITLE
Remove the vfio.sh utility file

### PR DESCRIPTION
**Pull request by fedepaol**
**What this PR does / why we need it**:
Since we now rely on the [OpenShift SR-IOV operator](https://github.com/openshift/sriov-network-operator) to perform all the heavylifting related to the configuration, there is no need of the file anymore.

Moreover, in this PR we align the local doc file mentioning the operator.
**Release note**:

```release-note

NONE

```

### CODE REVIEW

**Comment by fedepaol** [comment]
cc @booxter

**Comment by rmohr** [comment]
/lgtm

to the code change.

File: docs/sriov.md [hunk 1]
```diff
@@ -20,35 +20,16 @@ pci=realloc
 pci=assign-busses
 ```
 
-If all goes well, after reboot you should be able to enable SR-IOV VFs for
-capable NICs.
+If all goes well, after reboot your SR-IOV capable NICs should be ready to be used.
```
Comment (booxter) [inline docs/sriov.md]: -> should be ready to use

File: docs/sriov.md [hunk 2]
```diff
@@ -20,35 +20,16 @@ pci=realloc
...
 
-To enable VFs, you should do the following:
+
+Check for sr-iov devices by doing the following:
```
Comment (booxter) [inline docs/sriov.md]: SR-IOV, upper case

**Pull request merged**